.travis.yml

```diff
@@ -1,5 +1,14 @@
-language: node_js
-node_js: stable
-sudo: false
-notifications:
-  email: false
+language: node_js
+node_js: stable
+sudo: false
+notifications:
+  email: false
+deploy:
+  provider: npm
+  email: mdn-npm@mozilla.com
+  api_key:
+    secure: tPV4Rve++cmJeYOEhB0/bYaUVIxx9fyKm3Nn4mEBoTuqUojxCzjYaRNqhCREUigQayKDd6Efa6tzHbTygyU41WUNRLtRYENqv060nj8Bdzx7bjgm8dRqdctrvgisfqBfuQa+5MbliZblhD6M7ZyoURqK2stY64JGhvonKG1w0zl2sVyhDNSzoJwx6dx57upkVvXFqk4PQtmtLxRc27YlEiPa7yUCkXYUVCrQVQst7iJfR3pOvvfsR3lhyCJqVrV7cFgTy1VmpxHK/PjbVMJx4JahLbDHbS1I+jRWwx8/aPKMuQfoDn+34Y6uqVe+NaAwoy7bppsKOx/hpAY9HNfLBjIk8oe2n1JXmQg9J8bQ4sIijn1uflcly+LD2FYJSqtkHSsTSTiGHSH6+0O79tYcFRp3E62Xq+ED4Bx4h44Il4W9Z9ggrU4Lcp5BzNGgwf76U6V/Gx/vgd4BmiyDV/7+1MrHJxM67P/5rNAD+fdYQ6Hh5d7765ljBT4ttokoyhgKhEA5ShjPVt+ubotbgJ7ixY72O926qEAvQ7n74dEd4vnGXp2cO/h2dEPAstENY1xeuMyT4fz2CZ/zCr6xEZ9vkevPSFXoHdk2BzrFKLhelBZw9VqmD5RSwhR4/6dB4VxMWB79QACdHslMCWkFVgiFWTuAa4Prmd7J6E5B03dtHd4=
+  on:
+    tags: true
+    branch: master
```

Comment:
I believe ``branch: master`` says to release with each merge to master. Please remove this line and just leave the ``tags`` one.

Reply:
Thanks, I think I misunderstood the docs.